static/src/js/utils/cmrKeywords.js

```diff
@@ -0,0 +1,191 @@
+import { cloneDeep } from 'lodash-es'
```

Comment:
This file can be remove. We have already moved all of these into individual util functions.